ADALiOS/ADALiOS/ADBrokerKeyHelper.m

```diff
@@ -23,7 +23,7 @@
 #import "ADKeyChainHelper.h"
 #import <CommonCrypto/CommonCryptor.h>
 #import <Security/Security.h>
-#import "ADLogger.h"
+#import "ADLogger+Internal.h"
```

Comment:
Just out of curiosity, is it like a naming convention in objc to have the "+" in the name?